roles/kubernetes/master/templates/kube-scheduler-policy.yaml.j2

```diff
@@ -10,7 +10,6 @@
     {"name" : "GeneralPredicates"},
     {"name" : "CheckNodeMemoryPressure"},
     {"name" : "CheckNodeDiskPressure"},
-    {"name" : "CheckNodePIDPressure"},
```

Comment:
@Miouge1 Maybe you should just add a conditional around `kube_version` var for all the ones added.

Reply:
I think this is v1.12.0-alpha.0